app/views/assignments/_form.html.erb

```diff
@@ -11,61 +11,48 @@
   <% end %>
   <%= hidden_field_tag "which_form", (@assignment.id ? "update_assignment" : "new_assignment"), { :id => "which-form" } %>
   <div class="row">
-  <p> Hover over ⓘ for more details about any terminology.</p>
```

Comment:
Not sure why diff generates this block as a difference in code.  It looks identical.